doc/man3/SSL_CTX_set_tmp_dh_callback.pod

```diff
@@ -63,12 +63,11 @@ openssl L<openssl-dhparam(1)> application. This application
 guarantees that "strong" primes are used.
 
 Files dh2048.pem, and dh4096.pem in the 'apps' directory of the current
-version of the OpenSSL distribution contain the 'SKIP' DH parameters,
-which use safe primes and were generated verifiably pseudo-randomly.
-These files can be converted into C code using the B<-C> option of the
-L<openssl-dhparam(1)> application. Generation of custom DH
-parameters during installation should still be preferred to stop an
-attacker from specializing on a commonly used group. File dh1024.pem
+version of the OpenSSL distribution contain two of the MODP Diffie-Hellman
+groups for IKE as per RFC 3526.  These files can be converted into C code
+using the B<-C> option of the L<openssl-dhparam(1)> application. Generation
+of custom DH parameters during installation should still be preferred to
+stop an attacker from specializing on a commonly used group. File dh1024.pem
 contains old parameters that must not be used by applications.
```

Comment:
Why do we even have this paragraph in this manual?  Are we seriously telling people that they should go fetch the OpenSSL source to find some random files, *in the manual for `SSL_set_tmp_dh()` and friends???*

If we're serious about it, we should probably *install* those files, in something like $OPENSSLDIR/misc

Reply:
Side note:

``` console
$ find . -name dh2048.pem
./apps/dh2048.pem
./crypto/dh/dh2048.pem
$ find . -name dh4096.pem
./apps/dh4096.pem
./crypto/dh/dh4096.pem
```

Reply:
I didn't notice the second copy.

Reply:
I agree that the manual page needs a major overhaul, my intention here was to correct a blatant factual error that took me way too long to figure out....

Reply:
The PEMs in question are not even meant to be used for TLS....

Reply:
If we're not installing them, then this paragraph shouldn't be here at all

Reply:
I can do this.  I'll see if I can remove the files at the same time -- there is no point us having non-TLS DH parameters hanging around.

Reply:
Uhmmm, I believe that the one in apps is used in some tests, so that would require rewriting them. I suggest doing that as a separate PR.

Reply:
I've removed all three of the .pem files and the paragraph from the documentation.